.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
+    	"flowtype.flow-for-vscode",
+        "ms-vscode.js-atom-grammar"
```

Comment:
Are you sure the plugin `js-atom-grammar` is requirement for us? 

Looking at this plugin doc, I see:

> Since the latest update, the JavaScript grammar shipped with VS Code now also supports colorizing of varable and function references and aims to be feature equivalent to the Atom grammar. The VS Code team therefore recommends to use the built-in JavaScript grammar.

Reply:
Ok just noticed the "Known issue" on the Flow plugin, but it seems to be an old comment https://github.com/flowtype/flow-for-vscode/commit/aa5656a0dbd34efc5a7b61aaf989e3d73f3ad1d7

I don't have "ms-vscode.js-atom-grammar" installed locally and never had any issue with Flow syntax.